IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -123,7 +123,37 @@ partial model RectangularZoneTemplateInterface
       tab="Ceiling",
       group="Window details",
       enable=hasWinCei));
-
+  
+  parameter Modelica.Units.SI.Length h_winA = min(2, sqrt(A_winA))
+    "Window A height, including frame"
```

Comment:
This formulation was changed to `h_winA(min=0.1)=max(0.1,sqrt(A_winA))` to avoid division by 0 (which happens in RectangularZoneTemplate if there is no window, because then `A_winA`. See commit f9fc0e959 for the fix (not pushed to this branch yet).